From 0fac3f11205c97818fb36ac9be6f2e72ec67f37e Mon Sep 17 00:00:00 2001
From: Neil Roberts <neil@linux.intel.com>
Date: Wed, 3 Aug 2011 14:50:02 +0100
Subject: [PATCH 3/3] configure.ac: Add -no-undefined to the libtool flags

This flag lets libtool know that the library doesn't have any
undefined symbols that will be resolved before they are used. This
should be the case for Mx. The flag is required to build a DLL on
Windows because the feature isn't supported.
---
 configure.ac |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index e03dffe..6bd9a6b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -52,7 +52,7 @@ MX_LT_CURRENT=lt_current
 MX_LT_REV=lt_revision
 MX_LT_AGE=lt_age
 MX_LT_VERSION="$MX_LT_CURRENT:$MX_LT_REV:$MX_LT_AGE"
-MX_LT_LDFLAGS="-version-info $MX_LT_VERSION"
+MX_LT_LDFLAGS="-no-undefined -version-info $MX_LT_VERSION"
 
 AC_SUBST(MX_LT_VERSION)
 AC_SUBST(MX_LT_LDFLAGS)
-- 
1.7.3.16.g9464b

